### PR TITLE
Docs/known issues

### PR DESCRIPTION
**Pull request by oblm**
**Contributions**: We are not currently taking public contributions - see our [contributions](https://docs.improbable.io/unity/alpha/contributing) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Clarify known issue and fixes. This contains all of the changes in the now abandones https://github.com/spatialos/gdk-for-unity/pull/539, and addresses all requested changes in that PR.
#### Documentation
Rendered markdown.
#### Primary reviewers
@ElleEss

### CODE REVIEW

File: docs/content/get-started/open-project.md [hunk 1]
```diff
@@ -5,25 +5,23 @@ Launch the Unity Editor. It should automatically detect the project but if it do
...
-2. In the Project panel, navigate to **Assets** > **Fps** > **Art** > **Materials**.
-3. Right click on `Source_Shaders` and press Reimport.
+2. In the Project window, navigate to **Assets** > **Fps** > **Art** > **Materials**.
+3. Right click on the `Source_Shaders` directory and select Reimport.
 
 <img src="{{assetRoot}}assets/shader-fix.png" style="margin: 0 auto; display: block;" />
 
 #### Bake Navmesh
-There is a bug where the Unity Editor does not import the navmesh for the `FPS-SimulatedPlayerCoordinator` correctly when opening a project for the first time. To fix this, you need to rebake the navmesh for this scene.
+There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor that stops it importing the navmesh for the `FPS-SimulatedPlayerCoordinator` correctly when you open the project for the first time. To fix this, you need to rebake the navmesh for this Scene. To do this:
```
Comment (jessicafalk) [inline docs/content/get-started/open-project.md]: `stops it importing` sounds slightly off
maybe: 

There is a known issue in the Unity Editor regarding importing a navmesh. The navmesh for the `FPS-SimulatedPlayerCoordinator` is not imported correctly when opening the project for the first time.

File: docs/content/get-started/open-project.md [hunk 2]
```diff
@@ -5,25 +5,23 @@ Launch the Unity Editor. It should automatically detect the project but if it do
...
-There is a bug in the current preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended.
+There is a [known issue]({{urlRoot}}/known-issues) in the preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended.
 
 There is a quick fix however:
 
 1. Open the FPS Starter Project in the Unity Editor located in `workers/unity`.
-2. In the Project panel, navigate to **Assets** > **Fps** > **Art** > **Materials**.
-3. Right click on `Source_Shaders` and press Reimport.
+2. In the Project window, navigate to **Assets** > **Fps** > **Art** > **Materials**.
+3. Right click on the `Source_Shaders` directory and select Reimport.
```
Comment (jessicafalk) [inline docs/content/get-started/open-project.md]: should it be
1.
2.
3.

or 
1.
1.
1.
?

File: docs/known-issues.md [hunk 3]
```diff
@@ -13,3 +13,5 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
...
 | [Reader [Require] fields]({{urlRoot}}/content/gameobject/reading-and-writing-component-data.md#how-to-react-to-component-property-changes): `Reader.<component property name>Updated` may be invoked even if `<component property name>` did not change.                                                                                 | 2018/09/25 | None         | Verify manually whether a component property has actually changed.    | No     |
+| There is a bug in the preview version of the [High Definition Render Pipeline (Unity Blog)](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended. | 2018/10/12 | [566245](https://forum.unity.com/threads/shadergraph-subgraph-importing-bug-on-shared-project.566245/)         | Manually reimport shaders.                                                                  | No     |
+| The Unity Editor doesn't correctly import navmeshes. | 2018/10/12 | None         | None                                                                  | No     |
```
Comment (jessicafalk) [inline docs/known-issues.md]: not correct. The workaround is to regenerate the navmesh after opening the project

**Comment by oblm** [comment]
Thanks for the review @jessicafalk, I've fixed all of the defects you noted.

**Pull request merged**